### PR TITLE
Dropdown Component Added

### PR DESCRIPTION
**Pull request by BSubashRaj**
I Have Created a Dropdown Component Madhu can you check if its proper.

### CODE REVIEW

File: Lpicks-Core/src/app/core/dropdown/dropdown.component.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit, Input, ViewChild, ElementRef, HostListener } from '@angular/core';
+
+@Component({
+  // tslint:disable-next-line: component-selector
+  selector: 'kdi-dropdown',
+  templateUrl: './dropdown.component.html',
+  styleUrls: ['./dropdown.component.scss']
+  // host: { '(document:click)': 'handleClick($event)' }
```
Comment (Madhusuthanan-B) [inline Lpicks-Core/src/app/core/dropdown/dropdown.component.ts]: Remove commented code

File: Lpicks-Core/src/app/core/dropdown/dropdown.component.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+})
+export class DropdownComponent implements OnInit {
+  open = false;
+  @Input() dropdown: { title: string, items: []};
```
Comment (Madhusuthanan-B) [inline Lpicks-Core/src/app/core/dropdown/dropdown.component.ts]: The type { title: string, Items: []} needs to be moved into an interface. and use that here.

File: Lpicks-Core/src/app/core/dropdown/dropdown.module.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { DropdownComponent } from './dropdown.component';
+import { ClickOutsideDirective } from './clickOutside.directive';
+
+@NgModule({
+  declarations: [
+    DropdownComponent,
+    ClickOutsideDirective
```
Comment (Madhusuthanan-B) [inline Lpicks-Core/src/app/core/dropdown/dropdown.module.ts]: This is good. The click outside itselft could be used by many other components/modules later. Can you make clickoutside as a module and use that module here?

File: Lpicks-Core/src/assets/scripts/js/helpers.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+var getClosest = function ( elem, selector ) {
```
Comment (Madhusuthanan-B) [inline Lpicks-Core/src/assets/scripts/js/helpers.js]: What is the purpose of this file?

File: Lpicks-Core/src/assets/scripts/js/navigation.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
+var navigation = (function() {
```
Comment (Madhusuthanan-B) [inline Lpicks-Core/src/assets/scripts/js/navigation.js]: I see many js and sass files added. Please remove if they are not required.

File: Lpicks-Core/src/assets/scripts/js/helpers.js (hunk 4 above)
Comment (BSubashRaj) [reply Lpicks-Core/src/assets/scripts/js/helpers.js]: I copied the whole assets folder.

**Comment by Madhusuthanan-B** [comment]
There should not be 400+ files merged in a single PR. Remove all unncessary files.

**Pull request merged**